vast/integration/reference/arrow-export/step_01.ref

```diff
@@ -1,30 +1,37 @@
+    -- field metadata --
+    -- field metadata --
+    -- field metadata --
+    -- field metadata --
+    ARROW:extension:metadata: 'vast.address'
+    ARROW:extension:metadata: 'vast.address'
+    ARROW:extension:name: 'vast.address'
+    ARROW:extension:name: 'vast.address'
+    VAST:name:0: 'port'
+    VAST:name:0: 'port'
   -- field metadata --
   -- field metadata --
   -- field metadata --
-  -- field metadata --
-  -- field metadata --
-  -- field metadata --
-  ARROW:extension:metadata: 'vast.address'
-  ARROW:extension:metadata: 'vast.address'
-  ARROW:extension:name: 'vast.address'
-  ARROW:extension:name: 'vast.address'
   VAST:attributes:0: '{ "index": "hash" }'
-  VAST:name:0: 'port'
-  VAST:name:0: 'port'
   VAST:name:0: 'timestamp'
+  VAST:name:0: 'zeek.conn_id'
   child 0, item: string
+  child 0, orig_h: fixed_size_binary[16]
+  child 1, orig_p: uint64
+  child 2, resp_h: fixed_size_binary[16]
+  child 3, resp_p: uint64
 -- schema metadata --
 VAST:name:0: 'zeek.conn'
+_path: string
```

Comment:
Note to self: I think it's okay to effectively duplicate the type name here, as it preserves the of the input. In fact, it's probably a highly desirable property in case we need to convince someone that we really don't throw away anything, i.e., the original output can be reconstructed if need be.